src/fosslight_dependency/_package_manager.py

```
@@ -475,57 +494,88 @@ def add_allDeps_in_gradle(self, gradle_file):
+    config = android_config if self.package_manager_name == const.ANDROID else gradle_config
+
+    try:
+        for stack, name in create_dep_stack(self, f_name, config):
+            self.total_dep_list.append(name)
+            if len(stack) == 0:
+                self.direct_dep_list.append(name)
+            else:
+                if stack[-1] not in self.relation_tree:
+                    self.relation_tree[stack[-1]] = []
+                self.relation_tree[stack[-1]].append(name)
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)